src/components/SQForm/SQFormMaskedTextField.tsx

```diff
@@ -1,14 +1,52 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import {TextFieldProps, InputProps} from '@material-ui/core';
 import MaskedInput from 'react-text-mask';
+import BaseFieldProps from 'types/BaseFieldProps';
+import {maskProp} from '../../types/MaskTypes';
 import SQFormTextField from './SQFormTextField';
 
-function TextFieldMask({inputRef, mask, ...rest}) {
+interface SQFormMaskedTextFieldProps extends BaseFieldProps {
+  /** Valid mask array; custom or from utils/masks.js */
+  mask?: maskProp;
```

Comment:
Shouldn't `mask` be required? I know it wasn't before, but if the consumer doesn't provide a mask they should just use a TextField, right?

Reply:
I figure that maybe there'd be a use case for a conditional mask and instead of conditionally using a whole different component they can just not supply the mask? 🤷

Reply:
Yeah, that makes sense